ragtime.jdbc/src/ragtime/jdbc.clj

```diff
@@ -33,7 +41,6 @@
     (sql/insert! db-spec migrations-table
                  {:id         (str id)
                   :created_at (format-datetime (Date.))}))
-  
```

Comment:
Removed. Though fwiw whitespace on a newline is nice to not have hanging around.